Update "Reset the password" instructions for linux

halter73 opened the pull request:
I tested the updated command with the azure cloud shell. Without a resource group specified, I got the following error:

```
"+ Looking up the VM "VMAccessForLinux"
error:   Resource group 'myVM' could not be found.
error:   Error information has been recorded to /home/myName/.azure/azure.err
error:   vm extension set command failed"
```

Without changing "1.*" to "1.4", I got:
```
info:    Executing command vm extension set
+ Looking up the VM "myVM"
+ Installing extension "VMAccessForLinux", VM: "myVM"
error:   The value of parameter typeHandlerVersion is invalid.
error:   Error information has been recorded to /home/myName/.azure/azure.err
error:   vm extension set command failed
```

After my suggested edits, the command has been stuck at the following stage for at least 10 minutes:
```
info:    Executing command vm extension set
+ Looking up the VM "myVM"
- Installing extension "VMAccessForLinux", VM: "myVM"
```

Someone should probably look into this further to see if it's still even possible to reset a password on a linux VM. The other sections probably also need similar updates.

azurecla commented:
Hi __@martijnsublime__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

cjgronlund commented:
@martijnsublime 
Thank you for this contribution. I'm accepting this pull request.

--Carolyn
Azure Documentation

halter73 committed:
Update "Reset the password" instructions for linux

- I tested the updated command with the azure cloud shell
- Without a resource group specified, I got the following error:
"+ Looking up the VM "VMAccessForLinux"
error:   Resource group 'myVM' could not be found.
error:   Error information has been recorded to /home/myName/.azure/azure.err
error:   vm extension set command failed"

- Without changing "1.*" to "1.4", I got:
info:    Executing command vm extension set
+ Looking up the VM "myVM"
+ Installing extension "VMAccessForLinux", VM: "myVM"
error:   The value of parameter typeHandlerVersion is invalid.
error:   Error information has been recorded to /home/myName/.azure/azure.err
error:   vm extension set command failed

- After my suggested edits, it now seems to be permanently stuck at the following stage:
info:    Executing command vm extension set
+ Looking up the VM "myVM"
- Installing extension "VMAccessForLinux", VM: "myVM"

- So someone should probably look into this further to see if it's still even possible to reset a password on a linux VM. The other sections probably also need similar updates.
